.github/workflows/ci.yml

```diff
@@ -10,17 +10,17 @@ on:
 
 env:
   # Common versions
-  GO_VERSION: '1.19.5'
-  GOLANGCI_VERSION: 'v1.49.0'
-  DOCKER_BUILDX_VERSION: 'v0.10.0'
+  GO_VERSION: "1.19.5"
+  GOLANGCI_VERSION: "v1.49.0"
+  DOCKER_BUILDX_VERSION: "v0.10.0"
```

Comment:
any particular reason for this change?

Reply:
Accidentally turned on global format-on-save in my editor. Reverted.